ecosystem/sep-0006.md

```diff
@@ -114,13 +115,9 @@ SEP-6 lays out many options for how deposit and withdrawal can work. These are r
 * Decide which endpoints, if any, need to be [authenticated](#authentication), and declare that properly in the `/info` endpoint.
 * Pick your approach to [fees](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md#fee). We recommend using `/info` to express fees as it provides a better user experience (the user can see the fee structure in the wallet early in the process).
 * **For both deposit and withdrawal**:
-    * We recommend you use the [interactive flow](#3-customer-information-needed-interactive). It's flexible and simple to integrate.
-    * To start an interactive flow, provide the [Customer info needed response](#3-customer-information-needed-interactive).
+    * The interactive flow has been deprecated, if you require an interactive flow use [SEP-24](sep-0024.md).  The following details are to understand legacy interactive flows.
     * Include the `id` field in your response so the wallet can check up on the status of the transaction if it wants.
     * Also include the `id` field in the popup URL you provide to the wallet. This allows you to keep track of the transaction  when the user visits the URL.
-    * Make sure to pass along the `amount` to the popup URL, especially on the `/withdraw` flow.
-    * We recommend you use [SEP-10 authentication](#authentication) in the interactive flow and do not separately prompt for password to achieve a good user experience (although asking for MFA when confirming a transaction or requiring email confirmation is reasonable). Be sure to accept the [`jwt` parameter](#adding-parameters-to-the-url) so the user does not have to re-login. Support the `callback` parameter as well.
```

Comment:
Don't we want to keep this for the case where a client wants to use an interactive SEP-6 anchor? It seems like removing this would just require the client to contact the anchor to understand how to authenticate.